### ISSUE TITLE
[1.18.2] play_once broken (again?!)

### ISSUE THREAD

**Issue by Endgineer** [TheComputerizer/Music-Triggers#137]
**Describe the bug**
Music plays as expected. The only thing that doesn't work as expected is the music plays again when it ends. What I want to do is have the music play only once per session, when triggered, and never again until the next session.

**To Reproduce**
Refer to the `main.toml` file below. Ignore the numbers (I extended world depth); you can still test the `layer1` trigger by going anywhere below `y = 0`.
```
[triggers]
	[[triggers.zones]]
		identifier = "layer1"
		zone_min_y = "-288"
		zone_max_y = "0"
	[[triggers.zones]]
		identifier = "layer2"
		zone_min_y = "-576"
		zone_max_y = "-288"
	[[triggers.zones]]
		identifier = "layer3"
		zone_min_y = "-864"
		zone_max_y = "-576"
	[[triggers.zones]]
		identifier = "layer4"
		zone_min_y = "-1152"
		zone_max_y = "-864"
	[[triggers.zones]]
		identifier = "layer5"
		zone_min_y = "-1440"
		zone_max_y = "-1152"
	[[triggers.zones]]
		identifier = "layer6"
		zone_min_y = "-1728"
		zone_max_y = "-1440"
[songs]
	[songs.layer1]
		play_once = "4"
		must_finish = true
		triggers = [ "zones-layer1" ]
	[songs.layer2]
		play_once = "4"
		must_finish = true
		triggers = [ "zones-layer2" ]
	[songs.layer3]
		play_once = "4"
		must_finish = true
		triggers = [ "zones-layer3" ]
	[songs.layer4]
		play_once = "4"
		must_finish = true
		triggers = [ "zones-layer4" ]
	[songs.layer5]
		play_once = "4"
		must_finish = true
		triggers = [ "zones-layer5" ]
	[songs.layer6]
		play_once = "4"
		must_finish = true
		triggers = [ "zones-layer6" ]
```

**Additional context**
Using `musictriggers-1.18.2-6.2`, the version where this issue was presumed fixed. I tested both `play_once = "3"` and `play_once = "4"`.

**Comment by TheComputerizer** [comment]
play_once 3-5 haven't been properly implemented as of 6.2, though the GUI description was updated ahead of time. They have since been implemented in 6.3 which is slated to release soon

**Comment by TheComputerizer** [comment]
Fixed in 6.3
